helpers/util.js

```diff
@@ -0,0 +1,13 @@
+export function isNotNullOrUndefined(e) {
+	return e != null;
+}
+
+export function isNullOrUndefined(e) {
+	return e == null;
+}
```

Comment:
Ik zie het nut van zo'n util in sommige programmeertalen, maar in javascript ontgaat het mij wat. Als je het leesbaarder vindt, ga ik er ook niet over vallen, maar de inverse ook nog eens expliciet als methode maken voelt mij als developer wel heel vreemd aan.